packages/lib/src/wasm/wasm.js

```diff
@@ -400,16 +400,19 @@ function finalizeInit(instance, module) {
     return wasm;
 }
 
-function initSync(module, maybe_memory) {
+async function initSync(module, maybe_memory) {
```

Comment:
is this method idempotent? can't quite tell. if not, should we have logic in the calling function not to double initialize?

Reply:
@lsankar4033  It is idempotent but yeah, better to initialize it only once for performance.
Modified here: 9ddf979a462313ccc83d37f5f406c1c65701c9ad